guides/common/modules/proc_installing-standalone-hammer.adoc

```diff
@@ -3,11 +3,34 @@
 
 You can install Hammer on a host running {install-on-os} that has no {ProjectServer} installed, and use it to connect from the host to a remote {Project}.
 
+.Prerequisites
 .Prerequisites
```

Comment:
Does this mean a duplicate?

Reply:
It looks like it although it isn't showing in the preview. In any case, good catch, I'm dropping the extra line.